scripts/add_cluster.sh

```
@@ -0,0 +1,154 @@
+            --profile $AWS_PROFILE
+    else
+        echo "Node group 'observability' already exists, skipping creation."
+    fi
+
+    gum spin --spinner dot --title "Setting current context to $CLUSTER..." --show-error -- \
+    aws eks update-kubeconfig --name $CLUSTER --region=$REGION --profile $AWS_PROFILE
+fi
+
+# Login to ArgoCD
+gum spin --spinner dot --title "Logging in to ArgoCD..." --show-error -- \
+argocd login $ARGOCD_SERVER --username admin --password $(kubectl get secret argocd-initial-admin-secret -n argocd -o jsonpath="{.data.password}" --context $CTRL_PLANE_CTX | base64 --decode) --insecure --plaintext
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)